### PR TITLE
Add unit tests for LogManager and LogWriter

### PR DESCRIPTION
**Pull request by henryre**
See title.

### CODE REVIEW

**Comment by bhancock8** [comment]
Ping when tests pass.

**Comment by henryre** [comment]
@bhancock8 passing!

File: test/classification/training/loggers/test_log_writer.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import json
+import os
+import shutil
+import tempfile
+import unittest
+
+from snorkel.classification.training import LogWriter
+
+
+class TestLogManager(unittest.TestCase):
```
Comment (bhancock8) [inline test/classification/training/loggers/test_log_writer.py]: TestLogWriter

File: test/classification/training/loggers/test_log_manager.py [hunk 2]
```diff
@@ -1,31 +1,26 @@
...
     def test_log_manager_points(self) -> None:
         """Unit test of log_manager (points)"""
         log_manager_config = {"counter_unit": "points", "evaluation_freq": 10}
 
         checkpointer = Checkpointer(
-            checkpoint_dir=TEST_LOG_DIR, checkpoint_factor=2, **log_manager_config
+            checkpoint_dir=self.test_dir, checkpoint_factor=2, **log_manager_config
         )
-        log_manager = LogManager(
+        log_manager = LogManager(  # type: ignore
```
Comment (bhancock8) [inline test/classification/training/loggers/test_log_manager.py]: Is it the kwargs that make this not type checkable?

Comment (henryre) [reply test/classification/training/loggers/test_log_manager.py]: yes

File: test/classification/training/loggers/test_log_writer.py (hunk 1 above)
Comment (henryre) [reply test/classification/training/loggers/test_log_writer.py]: will merge then test after

Comment (henryre) [reply test/classification/training/loggers/test_log_writer.py]: fix

**Pull request merged**